presto-hive/src/main/java/com/facebook/presto/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -79,4 +80,17 @@ public GlueHiveMetastoreConfig setDefaultWarehouseDir(String defaultWarehouseDir
         this.defaultWarehouseDir = Optional.ofNullable(defaultWarehouseDir);
         return this;
     }
+
+    public Optional<String> getIamRole()
+    {
+        return iamRole;
+    }
+
+    @Config("hive.metastore.glue.iam-role")
```

Comment:
Looks like we don't have glue metastore documentation, would be nice to add docs for glue metastore & document this config. I will appreciate if you can do it as a follow up PR.